### ISSUE TITLE
[Bug] [Crash] Can't upgrade Numpy in Stable Diffusion WebUI and Stable Diffusion Forge WebUI in Stability Matrix

### ISSUE THREAD

**Issue by Giribot** [LykosAI/StabilityMatrix#610]
Hello !

![Numpy](https://github.com/LykosAI/StabilityMatrix/assets/29394604/cb8d86e0-42c2-45eb-ad25-cb2272975796)

In the tab: "Packages"
(choose "Stable Diffusion WebUI" OR "Stable Diffusion Forge  WebUI" ) => ((three dots) => "Python Packages" => numpy 1.26.4 => when i click on "Numpy", Stability Matrix crashes (RIP).
(but it's work on all others packages like "Foocus", (?))
**_(No problems for clicking the others Python Packages from "Accelerate" to "Numba" and after "Numpy" to the end of the list,  => only "Numpy" +++ crashes Stability Matrix. +++ )_**

_## Exception
ArgumentException: An item with the same key has already been added. Key: Name
### Sentry ID
```
748f59890f9e40c297fcd34fbfa4ee58
```
### Stack Trace
```
   at System.Collections.Generic.Dictionary`2.TryInsert(TKey key, TValue value, InsertionBehavior behavior)
   at System.Linq.Enumerable.ToDictionary[TSource,TKey,TElement](IEnumerable`1 source, Func`2 keySelector, Func`2 elementSelector, IEqualityComparer`1 comparer)
   at StabilityMatrix.Core.Python.PipShowResult.Parse(String output)
   at StabilityMatrix.Core.Python.PyVenvRunner.PipShow(String packageName)
   at StabilityMatrix.Avalonia.ViewModels.Dialogs.PythonPackagesItemViewModel.LoadExtraInfo(DirectoryPath venvPath)
   at StabilityMatrix.Avalonia.ViewModels.Dialogs.PythonPackagesItemViewModel.LoadExtraInfo(DirectoryPath venvPath)
   at AsyncAwaitBestPractices.SafeFireAndForgetExtensions.HandleSafeFireAndForget[TException](Task task, Boolean continueOnCapturedContext, Action`1 onException)
   at System.Threading.Tasks.Task.<>c.<ThrowAsync>b__128_0(Object state)
   at Avalonia.Threading.SendOrPostCallbackDispatcherOperation.InvokeCore()
   at Avalonia.Threading.DispatcherOperation.Execute()
   at Avalonia.Threading.Dispatcher.ExecuteJob(DispatcherOperation job)
   at Avalonia.Threading.Dispatcher.ExecuteJobsCore(Boolean fromExplicitBackgroundProcessingCallback)
   at Avalonia.Threading.Dispatcher.Signaled()
   at Avalonia.Win32.Win32Platform.WndProc(IntPtr hWnd, UInt32 msg, IntPtr wParam, IntPtr lParam)
   at Avalonia.Win32.Interop.UnmanagedMethods.DispatchMessage(MSG& lpmsg)
   at Avalonia.Win32.Win32DispatcherImpl.RunLoop(CancellationToken cancellationToken)
   at Avalonia.Threading.DispatcherFrame.Run(IControlledDispatcherImpl impl)
   at Avalonia.Threading.Dispatcher.PushFrame(DispatcherFrame frame)
   at Avalonia.Threading.Dispatcher.MainLoop(CancellationToken cancellationToken)
   at Avalonia.Controls.ApplicationLifetimes.ClassicDesktopStyleApplicationLifetime.Start(String[] args)
   at StabilityMatrix.Avalonia.Program.Main(String[] args)
```_



Thanks you !

**Comment by mohnjiles** [comment]
This should be fixed now with [v2.10.3](https://github.com/LykosAI/StabilityMatrix/releases/tag/v2.10.3)

**Comment by leakedby** [comment]
Got same error on 2.15.0
